templates/systemd-service_redis.template

```diff
@@ -8,6 +8,7 @@ TimeoutStartSec=infinity
 TimeoutStopSec=15
 Restart=always
 RestartSec=2s
+
 ExecStartPre=-/usr/bin/docker stop {{ service_redis_name }}
 ExecStartPre=-/usr/bin/docker rm {{ service_redis_name }}
 ExecStartPre=/usr/bin/docker pull {{ docker_redis_image }}
```

Comment:
Same as in the DB template: please create a variable for this command and make it work as in the main template (e.g., insert the line only if the variable is defined).